docs/update_api_list.py

```diff
@@ -117,7 +117,7 @@ def sort_key(obj):
 ]
 
 classes = find_all_classes(
-    (),
+    (object),
```

Comment:
Hah so we just weren't collecting any classes ever. This is a part of a documentation-building cookiecutter/skeleton/tooling that isn't super uniform, should we be adding this to all our "Python Library" packages?  🤔

Reply:
I'm wondering if this belongs as a standard tool in some form in e.g. pcdsutils or in a standalone module that we can pull in as a docs dependency.

Reply:
It's certainly a useful way to generate a full project API that is easier to browse than any more straightforward method in sphinx